tests/system/action/user/test_update.py

```diff
@@ -2436,6 +2436,7 @@ def test_group_removal_with_speaker(self) -> None:
                 "user/1234": {
                     "username": "username_abcdefgh123",
                     "meeting_user_ids": [4444, 5555],
+                    "is_present_in_meeting_ids": [4, 5],
```

Comment:
I expect an entry in some assert_model to check it. I don't see this here.
I also expect that the second part of the relation in the test.